docs/user-guide/generate-certificates.md

```diff
@@ -72,58 +89,81 @@ The following command output shows generation of a self signed PKCS12 certificat
 #>
 ```
 
-Because `--update-config` was specified the `zowe.certificates` section's values are updated to reference the newly generated certificate.  These updates are logged by the `zwe init certificate` command output.  Open the `zowe.yaml` file to check the references to the newly generated certificate values, as shown below:
+Because `--update-config` was specified the `zowe.certificates` section's values are updated to reference the newly generated certificate.  These updates are logged by the `zwe init certificate` command output. Open the `zowe.yaml` file to check the references to the newly generated certificate values, as shown below:
 
 ```
   certificate:
     keystore:
       type: PKCS12
-      file: /global/zowe/keystore/localhost/localhost.keystore.p12
+      file: /var/zowe/keystore/localhost/localhost.keystore.p12
       password: password
       alias: localhost
     truststore:
       type: PKCS12
-      file: /global/zowe/keystore/localhost/localhost.truststore.p12
+      file: /var/zowe/keystore/localhost/localhost.truststore.p12
       password: password
     pem:
-      key: /global/zowe/keystore/localhost/localhost.key
-      certificate: /global/zowe/keystore/localhost/localhost.cer
-      certificateAuthorities: /global/zowe/keystore/local_ca/local_ca.cer
+      key: /var/zowe/keystore/localhost/localhost.key
+      certificate: /var/zowe/keystore/localhost/localhost.cer
+      certificateAuthorities: /var/zowe/keystore/local_ca/local_ca.cer
 ```
 
-When using a self-signed certificate, you will be challenged by your browser when logging in to Zowe to accept its untrusted certificate authority.  Depending on the browser you are using there are different ways to proceed.
+If you want to double check the certificate you generate, run `keytool -v -list keystore localhost.keystore.p12` in the prompt for details about this certificate. See more commands to manage a keystore [here](https://docs.oracle.com/en/java/javase/11/tools/keytool.html).
 
-## Create a self signed JCERACFKS certificate
+### Next step
+
+When using a self-signed certificate, you will be challenged by your browser when logging in to Zowe to accept its untrusted certificate authority. Depending on the browser you are using there are different ways to proceed. See next step about how to import the PKCS12 certificate to your brower [here](./import-certificates#import-your-pkcs12-certificate).
+
+## Create a self-signed JCERACFKS certificate
+
+### Configure the JCERACFKS setup section in zowe.yaml
+
+For JCERACFKS certificate (z/OS keyring) users, here are some instructions about the fileds in the `zowe.yaml` file.
+
+<!-- - `zowe.setup.certificate.keyring.owner` is the keyring owner. It's optional and default value is `zowe.setup.security.users.zowe`. If it's also not defined, the default value is ZWESVUSR. Sam: didn't see "owner" in the yaml file. -->
```

Comment:
@Joe-Winchester Hi Joe, would you please help me confirm if there is a "owner" field in the corresponding section in `zowe.yaml`? Because I didn't see that in the [example yaml file](https://github.com/zowe/zowe-install-packaging/blob/v2.x/staging/example-zowe.yaml). Thank you very much!

Reply:
Hi @samanthasusu ,  good catch !  I have only ever created a keyring woth the owner default to the `zowe.setup.security.users.zowe` value of `ZWESVUSR`.  I didn't know it was possible to do a local override.
As a test I created 
```
zowe
   security:
      users:
        zowe: ZWESVUSR
...
    certificate:
      type: JCERACFKS
      createZosmfTrust: true
      keyring:
        # keyring name
        name: ZoweKeyring
        owner: WINCHJ
```
and ran the `zwe init certificate` to see what happened.  In the `ZWEKRING` JCL that was created it did use the `WINCHJ` value, 
```
//         SET  ZOWEUSER=WINCHJ
...
/* Create the keyring .............................................. */
   RACDCERT ADDRING(&ZOWERING.) ID(&ZOWEUSER.)
   SETROPTS RACLIST(DIGTRING) REFRESH
```
which confirms that the value is used in `zowe.setup.certificate.keyring.owner` if one is specified, so the doc is correct.
Thank you for pointing this out !!!

Reply:
Thank you @Joe-Winchester  for your explanation. That makes sense to me.